conftest.py

```diff
@@ -29,46 +31,44 @@ def pytest_configure(config):
     reporter = FixtureReporter(config)
     config.pluginmanager.register(reporter, 'fixturereporter')
 
+def _max_fname_len(file_system_path, default_max_len=80):
+    name_max_str = [s for s in os.pathconf_names if s.endswith('_NAME_MAX')]
+    if len(name_max_str) == 1:
+        try:
+            return os.pathconf(file_system_path, name_max_str[0])
+        except OSError:
+            pass
+    return default_max_len
+
+def _make_fname_valid(file_system_path, fname, len_margin):
```

Comment:
What's invalid about non-ascii filenames? Is there any reason this needs to be enforced?

Reply:
(if it does need to be enforced, we have [`util.file.string_to_file_name()`](https://github.com/broadinstitute/viral-ngs/blob/master/util/file.py#L571-L619))

Reply:
Actually the commented out portions of that function show that we made a point of allowing unicode filenames, we just disallowed certain pathsep or escape characters.